packages/front-end/components/AnalysisResultPopover/AnalysisResultPopover.tsx

```diff
@@ -394,6 +426,7 @@ export default function AnalysisResultPopover({
                     getFactTableById={
                       ssrPolyfills?.getFactTableById || getFactTableById
                     }
+                    asTd={false}
                   />
                 </TableCell>
               </TableRow>
```

Comment:
these shouldn't be here since they are now rendered in `<FlagCard>`:
```
      <Flex direction="column" gap="2">
        {maybeRenderLiftWarning()}
        {maybeRenderSuspiciousChange()}
      </Flex>
```